specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-07-19/quota.json

```diff
@@ -87,11 +87,11 @@
       },
       "put": {
         "tags": [
-          "QuotaRequests"
+          "Quota_CreateOrUpdate"
```

Comment:
Any reason why tag for same resource being different for different http method (QuotaInformation for get, QuotaRequest_Create for put)? Usually they are same.

Reply:
I will make thems ame.

Reply:
Updated to QuotaInformation .